cbor/shared/src/main/scala/fs2/data/cbor/Tags.scala

```diff
@@ -40,6 +40,6 @@ object Tags {
   final val RegularExpression: Long = 35L
   final val MimeMessage: Long = 36L
 
-  final val SelfDescribeCbor: Long = 55799L
+  final val SelfDescribedCbor: Long = 55799L
```

Comment:
This is bincompat-breaking if I'm not mistaken. We can fix that by making the old one package private while adding the new one. Still source breaking, but that's less of a concern.